1_fetch.R

```diff
@@ -107,19 +107,24 @@ p1_targets_list <- list(
         suppressWarnings()
       }
   ),
+  
+  tar_target(
+    p1_catchments_sf_valid, st_buffer(p1_catchments_sf,0)
+  ),
 
   # Download NLCD datasets 
   tar_target(
     p1_NLCD_data_zipped, 
-    download_NHD_NLCD_data(sb_id = sb_ids_NLCD,
+    download_NHD_data(sb_id = sb_ids_NLCD,
                            out_path = '1_fetch/out',
-                           downloaded_data_folder_name = NLCD_folders),
+                           downloaded_data_folder_name = NLCD_folders,
+                           output_data_parent_folder = 'NLCD_LC_Data'),
     format = 'file'),
   
   ## Unzip all NLCD downloaded datasets 
   ## Note - this returns a string or vector of strings of data path to unzipped datasets 
   tar_target(p1_NLCD_data_unzipped, 
```

Comment:
Can you change the formatting to match the other targets? New line after `tar_target(`